woocommerce/woocommerce-android
# Migrate CardReaderPaymentDialog to Hilt

This PR updates CardReaderPaymentDialog to Hilt. 

To test:
1. Open order detail screen
2. Tap on "Collect payment" button
3. Notice "Payment failed" dialog is shown and the app doesn't crash - it doesn't succeed since you are not connected to a reader

Update release notes:
